src/blockchain/wallet.ts

```diff
@@ -50,9 +52,45 @@ const listValidDepositTransactionsByWalletAddress = async (
   return [];
 };
 
+const getLockStatus = async (id: [BigNumber]): Promise<number> => {
+  const p2pContract = getContract();
+  const res = await p2pContract.getLocksStatus([id]);
+
+  return res[1][0];
+};
+
+const filterLockStatus = async (
+  transactions: Event[]
+): Promise<WalletTransaction[]> => {
+  const txs = [];
+
+  for (const transaction of transactions) {
```

Comment:
Don’t use iterators. Prefer JavaScript’s higher-order functions instead of loops like for-in or for-of. eslint

```suggestion
 const filteredTransactions = transactions.map(transaction -> {
    const walletTransaction : WalletTransaction = {...}
    return  walletTransaction;
 });
```

Reply:
Fixed, thx.